src/main/java/ca/nines/ise/cmd/Validate.java

```diff
@@ -75,6 +93,14 @@ public void execute(CommandLine cmd) throws Exception {
         if (dom.getStatus() != DOM.DOMStatus.ERROR) {
           dv.validate(dom, schema);
           nv.validate(dom);
+          scv.validate(dom);
+          sbv.validate(dom);
+          slv.validate(dom);
+          spv.validate(dom);
+          tbv.validate(dom);
+          hwv.validate(dom);
+          ov.validateDOM(dom);
+          rv.validateDOM(dom);
```

Comment:
the validateDOM() methods belong to the ruleValidator and OrnamentValidator. I haven't modified them. Should I change them to be in line with the rest?